src/foam/nanos/servlet/VirtualHostRoutingServlet.js

```diff
@@ -177,9 +177,6 @@ foam.CLASS({
       }
       // default fonts
       if ( headConfig == null || ! headConfig.containsKey("customFonts") || customFontsFailed ) {
-        out.println("<link href=\\"https://fonts.googleapis.com/css?family=Roboto:100,300,400,500\\" rel=\\"stylesheet\\">");
```

Comment:
Do we want to get of Roboto, since we use that in a lot of places:
https://github.com/kgrgreer/foam3/search?q=Roboto